source/elements/oneTBB/source/task_scheduler/task_arena/task_arena_cls.rst

```diff
@@ -27,11 +27,15 @@ A class that represents an explicit, user-managed task scheduler arena.
                 };
 
                 struct constraints {
-                    numa_node_id numa_node;
-                    int max_concurrency;
-
-                    constraints(numa_node_id numa_node_       = task_arena::automatic,
-                                int          max_concurrency_ = task_arena::automatic);
```

Comment:
@alexey-katranov @aleksei-fedotov 

Folks, how do you think, do we want to remove this constructor from the spec?

For me, it seems like the removing makes sense, but it breaks the source backward compatibility.

Reply:
It was decided that the constructor should exist for all pre-C++20 standards. C++20 standard and later shouldn't have this constructor.